src/page/Variant.tsx

```diff
@@ -357,6 +357,13 @@ class Variant extends React.Component<IVariantProps> {
                                 <Row>
                                     <Col>
                                         <BasicInfo
+                                            isIGV={
+                                                this.props.store.annotation
+                                                    .result
+                                                    ?.most_severe_consequence ===
```

Comment:
Using "most_severe_consequence" is good for now. Just a heads up that we are working on another fix to add  `IGR` to Variant_Classification and `intergenic_variant` to consequenceTerms in annotation_summary for intergenic variants, so we could use Variant_Classification === IGR when the new fields are available 
https://github.com/genome-nexus/genome-nexus/issues/725

Reply:
Sure, will update it asap.